commit af1e8878c005e468f8bcec1c17b1515f7473a496 (refs/changes/74/4674/3)
Author: Mike Gerdts <mike.gerdts@joyent.com>
Date:   2018-08-14T20:53:11+00:00 (1 year, 2 months ago)
    
    OS-7117 deliver uefi-rom.bin for bhyve
    Reviewed by: Jorge Schrauwen <jorge@blackdot.be>
    Reviewed by: Hans Rosenfeld <hans.rosenfeld@joyent.com>
    Reviewed by: Patrick Mooney <patrick.mooney@joyent.com>
    Approved by: Patrick Mooney <patrick.mooney@joyent.com>

diff --git a/overlay/generic/manifest b/overlay/generic/manifest
index d8b88125..ebc1a35c 100644
--- a/overlay/generic/manifest
+++ b/overlay/generic/manifest
@@ -177,6 +177,7 @@ f usr/sbin/zonememstat 0555 root bin
 f usr/sbin/zonemon 0555 root bin
 d usr/share/bhyve 0555 root bin
 f usr/share/bhyve/uefi-csm-rom.bin 0444 root bin
+f usr/share/bhyve/uefi-rom.bin 0444 root bin
 f usr/share/man/man1/json.1 0444 root bin
 f usr/share/man/man1m/fssstat.1m 0444 root bin
 d var/db 0755 root sys
diff --git a/overlay/generic/usr/share/bhyve/uefi-csm-rom.bin b/overlay/generic/usr/share/bhyve/uefi-csm-rom.bin
old mode 100755
new mode 100644
index 0cbe839c..0687c892
Binary files a/overlay/generic/usr/share/bhyve/uefi-csm-rom.bin and b/overlay/generic/usr/share/bhyve/uefi-csm-rom.bin differ
diff --git a/overlay/generic/usr/share/bhyve/uefi-rom.bin b/overlay/generic/usr/share/bhyve/uefi-rom.bin
new file mode 100644
index 00000000..cfefe0be
Binary files /dev/null and b/overlay/generic/usr/share/bhyve/uefi-rom.bin differ
